packages/vue-facebook-login-component/src/components/Scope.js

```diff
@@ -92,10 +92,10 @@ export default {
     },
     async login() {
       const promise = login(this.loginOptions)
-      const { status } = await this.async(promise)
-      if (Sdk.isConnected(status)) {
+      const result = await this.async(promise)
```

Comment:
Let's name it `response`.

Reply:
You should also run tests (they are not automated atm).

Reply:
Done.

NB: please add some notes [here](https://github.com/adi518/vue-facebook-login-component#development) about how to contribute, especially about running unit tests.

Reply:
Sure, good idea. 👍